apps/storybook.namekit.io/stories/Namekit/Input.stories.tsx

```diff
@@ -129,11 +134,20 @@ export const WithError: Story = {
   },
 };
 
-export const Disabled: Story = {
+export const PrimaryDisabled: Story = {
   args: {
     variant: "primary",
     inputSize: "medium",
-    value: "Disabled input",
+    value: "Primary disabled input",
+    disabled: true,
+  },
+};
+
+export const SecondaryDisabled: Story = {
+  args: {
+    variant: "secondary",
+    inputSize: "medium",
+    value: "Secondary disabled input",
```

Comment:
Please see my other comments about the problems of passing a `value` like this. This is an issue because in Storybook I can use the controls to make this UI component enabled again, but then it doesn't actually take input in this story. For a quick fix, suggest switching to a placeholder value.